MEMBERSHIP.md

```diff
@@ -0,0 +1,51 @@
+
+## Ambassador
+
+Gli ambassador dimostrano di aderire ai principi di questo documento, di avere familiarità con l'organizzazione del progetto, i ruoli, le politiche, le procedure, le convenzioni, ecc. Le aspettative, le responsabilità e i requisiti specifici dei ruoli sono elencati di seguito.
+
+# Ambassador
+
+Gli ambassador sono membri costantemente attivi della community. Questi sono tenuti a rimanere attivi nella community per mantenere il livello.
+
+**Definito da**: Contributor dell'organizzazione GitHub di Il Libro Open Source e l'ingresso nella chat Telegram della community dedicata agli Ambassador.
+
+## Requisiti
+
+- Abilitazione dell'autenticazione a due fattori sul proprio account GitHub
+- Hanno apportato molteplici contributi ai progetti o alla community. Il contributo può includere, ma non è limitato a:
+  - Autorizzare o revisionare PR su GitHub
+  - Aiutare o commentare le issues su GitHub
+  - Contribuire al progetto o alle discussioni della community (e.g. riunioni, GitHub discussions)
+- Aver letto la linee guida per le community, le regole e il documento sugli Ambassador
+- Essere sponsorizzato da 2 (da valutare il numero) Ambassador attivi
+- Aprire un issue su questo repository utilizzando il template
+  - Assicuratevi che i vostri sponsor siano @menzionati sull'issue aperto così che possano rispondere
+  - Gli sponsor dovranno rispondere con un :+1: alla issue per accettare l'ingresso dell'utente come ambassador
+- Tutti gli ambassador saranno comunque liberi di intervenire nella discussione commentando e dando una loro personale visione che sia essa positiva o negativa
+
+## Responsabilità e privilegi
+
+- Rispondere ai problemi e alle relative PR o issue
+- Dare assistenza a nuovi membri della community
+- Contribuire attivamente alla scrittura di capitoli interni al libro Open Source
+- Poter proporsi come responsabile di uno o più capitoli del libro Open Source
+- Poter proporsi all'interno di uno dei working group
+- Gli Ambassador possono approvare le PR aperte
+- Accesso al canale Telegram degli Ambassador
+
+# Ambassador inattivi
+
+_Gli ambassador sono membri costantemente attivi della community._
+
+Un principio fondamentale per mantenere una community sana è incoraggiare la partecipazione attiva. È inevitabile che gli interessi delle persone cambino nel tempo e non ci si aspetta che contribuiscano attivamente per sempre.
+
+Tuttavia, essere un Ambassador comporta una serie di permessi elevati. Questi permessi non dovrebbero essere utilizzati da chi non ha familiarità con lo stato attuale della community.
+
+Pertanto, gli Ambassador che si assentano per un lungo periodo dalla community senza svolgere alcuna attività al suo interno saranno rimossi e dovranno ripetere il processo di adesione dopo aver ripreso confidenza con lo stato attuale della community.
```

Comment:
```suggestion
```
Questo suggerisco di rimuoverlo. Riguarda gli ambassador e abbiamo altre linee guida a riguardo.

Reply:
Io lo manterrei, semplicemente per facilità di trovare tutte le informazioni di cui hai necessità in pochi, ma semplici file, se no il tutto diventa troppo frammentato e confusionario.

Questo file lo vedo come una descrizione di tutte le membership presenti nella Community e poi sta a te decidere cosa farci a riguardo!